app/Livewire/Forms/BackupForm.php

```
@@ -191,11 +192,22 @@ public static function rulesFor(
         array $entry,
         DatabaseType $serverType,
+        bool $isAgent,
     ): array {
         $prefix = "backups.{$index}.";
 
         $rules = [
-            $prefix.'volume_id' => 'required|exists:volumes,id',
+            $prefix.'volume_id' => [
+                'required',
+                'exists:volumes,id',
+                function (string $attribute, mixed $value, \Closure $fail) use ($isAgent): void {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$attribute'. (undefined)

(UnusedFormalParameter)